includes/stats/stats-importer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Wporg\TranslationEvents\Stats;
+
+use Wporg\TranslationEvents\Event\Event;
+
+class Stats_Importer {
+	/**
+	 * Imports the contributions a user made while a given event was active.
+	 */
+	public function import_for_user_and_event( int $user_id, Event $event ): void {
+		global $wpdb, $gp_table_prefix;
+
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
+		$wpdb->query(
+			$wpdb->prepare(
+				"
+				insert ignore into {$gp_table_prefix}event_actions
+				    (event_id, user_id, original_id, locale, action, happened_at)
+				select %d, t.user_id, t.original_id, ts.locale, %s, t.date_added
+				from {$gp_table_prefix}translations t,
+				     {$gp_table_prefix}translation_sets ts
+				where t.user_id = %d
+				  and t.translation_set_id = ts.id
+				  and ( t.status = 'current' or t.status = 'waiting' or t.status = 'changesrequested' or status = 'fuzzy' )
+				  and date_added >= %s
+				  AND date_added <= %s
```

Comment:
```suggestion
				  and date_added between %s and %s
```
https://dev.mysql.com/doc/refman/8.0/en/functions.html#operator_between
> this is equivalent to the expression (min <= expr AND expr <= max)

Reply:
Great suggestions, thanks!